roles-static/tsuru_api/tasks/main.yml

```diff
@@ -32,13 +32,17 @@
   wait_for: port={{api_port}} timeout=60
 
 # TODO: handle the case where the user already exists
-- name: Add admin user.
+- name: Add admin user team
   shell: >
     mongo tsuru --eval 'db.teams.update({_id: "admin"}, {_id: "admin"}, {upsert: true})';
     mongo tsuru --eval "db.teams.update({_id: 'admin'}, {\$addToSet: {users: '{{admin_user}}'}})";
```

Comment:
If we only get the exit code of the last command, should we split these into separate tasks to surface errors better?